src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -128,4 +131,19 @@ public function testObjectWithNullableConstructorArgument()
 
         $this->assertNull($dummy->getFoo());
     }
+
+    /**
+     * @requires PHP 7.1
```

Comment:
Shouldn't it be 5.6?

Reply:
You are right!

Reply:
After testing, it fails in 5.6 because I'm using `NullableConstructorArgumentDummy` (which requires PHP 7.1) as type hint in the fixture. It is not detected by travis-ci.org because PHP 5.6 is not being used to test PRs.

Reply:
Fixed using `Dummy` insteadof `NullableConstructorArgumentDummy`. Tests finally passes in PHP 5.6.